src/drivers/intel/cavs/ipc.c

```diff
@@ -143,7 +143,7 @@ static enum task_state ipc_platform_do_cmd(void *data)
 
 static void ipc_platform_complete_cmd(void *data)
 {
-#if CAVS_VERSION >= CAVS_VERSION_2_0
+#if defined CONFIG_SUECREEK
```

Comment:
Please don't use `defined` at all with configs, now it works because we have undefined configs for 'no' values, but they can be also falsy instead of undefined and then it'll stop working

Reply:
ACK